kod/object/active/holder/nomoveon/battler/monster/towns/kocatwn/kcbowmak.kod

```diff
@@ -27,6 +27,10 @@ resources:
          "in blood spilt in battle.  You will understand, then, that my price for them is high."
    KocatanBowMaker_unwanted_give = "~k%s%s tells you, \"I have enough of those type of supplies to hold us through the summer.~n~k\""
 
+   KocatanBowMaker_welcome_rsc = "If you've come seeking a fine bow, you've chosen well. "
+      "I craft each weapon to perfection - no idle chatter, no needless pleasantries. "
```

Comment:
Kind of ironic to hear idle chatter about how there's no idle chatter.  I think this would be stronger with this sentence removed.